account/wallet.go

```diff
@@ -0,0 +1,25 @@
+package account
```

Comment:
I would call this file `create_address.go` or `address.go` or put it into `utils.go`.

Reply:
Renamed!

Reply:
I still see `account/wallet.go` :D